gfx-memory/src/heaps/mod.rs

```diff
@@ -146,16 +146,13 @@ impl<B: hal::Backend> Heaps<B> {
             device,
             memory_index as u32,
             kind,
-            requirements.size,
-            requirements.alignment,
+            Size::new(requirements.size).unwrap(),
```

Comment:
should we then change gfx::memory::Requirements size/alignment to use NonZeroU64 as well?

Reply:
Yeah, that's what I've been considering as well. I'm thinking of trying some NonZero in gfx-hal and seeing how it goes.
It's ok to let this PR to sit for a bit.